hello.md

```diff
@@ -0,0 +1,152 @@
+# Hello Wasm
+
+Production C/C++ compilers like Emscripten/LLVM produce WebAssembly
+with extensive external bindings and standard libraries.
+When all of that is stripped away, a simple hello world program
+looks something like this:
+
+```
```

Comment:
For those not used to objdump, can you put a comment at the top explaining each column?